msteams-platform/task-modules-and-cards/cards/Universal-actions-for-adaptive-cards/Up-To-Date-Views.md

```diff
@@ -0,0 +1,241 @@
+---
+title: Up to date views
+description: Sample for up to date views using Universal Bot
+ms.topic: conceptual
+---
+
+## Up to date views
+Up to date views are now powered with a combination of Universal bots and bot fanouts in Teams.
+
+Example: A user creates an asset approval request in Teams conversation. Shiladitya creates an approval request and assigns it to Sowrabh and Dipesh. 
+
+Part 1: Now role based views can be leveraged by using refresh property of the adaptive cards.
+Using role based views one can show a card with Approve/Reject buttons to a set of users, and show a card without Approve/Reject buttons to other users depending on their role.
+
+Part 2: To keep the card state updated an all times, bot fanout mechanism can be leveraged.
```

Comment:
Let's use editing a message instead of "bot fanout" everywhere. Fanout is an internal implementation mechanism.

Reply:
done